lib/libspl/include/sys/isa_defs.h

```diff
@@ -100,28 +100,43 @@ extern "C" {
 
 #define	_SUNOS_VTOC_16
 
-/* arm arch specific defines */
-#elif defined(__arm) || defined(__arm__) || defined(__aarch64__)
+/* aarch64 arch specific defines */
+#elif defined(__aarch64__)
 
 #if !defined(__arm)
 #define	__arm
 #endif
 
-#if !defined(__arm__)
-#define	__arm__
-#endif
-
 #if defined(__aarch64__)
```

Comment:
this check can be removed now since we already have this block for `__aarch64__`.

Reply:
ok.